charts/ir-standalone/templates/store/statefulset.yaml

```diff
@@ -0,0 +1,210 @@
+    {{- if .Values.commonLabels }}
+    {{- include "graphscope-store.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "graphscope-store.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.store.replicaCount }}
+  selector:
+    matchLabels: {{ include "graphscope-store.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: store
+  serviceName: {{ include "graphscope-store.store.fullname" . }}-headless
+  updateStrategy:
+    type: {{ .Values.store.updateStrategy }}
+    {{- if (eq "Recreate" .Values.store.updateStrategy) }}
+    rollingUpdate: null
+    {{- end }}
+  template:
+    metadata:
+      annotations:
+        {{- if .Values.store.podAnnotations }}
+        {{- include "graphscope-store.tplvalues.render" (dict "value" .Values.store.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+      labels: {{- include "graphscope-store.labels" . | nindent 8 }}
+        app.kubernetes.io/component: store
+        {{- if .Values.commonLabels }}
+        {{- include "graphscope-store.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "graphscope-store.serviceAccountName" . }}
+      {{- if .Values.store.affinity }}
+      affinity: {{- include "graphscope-store.tplvalues.render" (dict "value" .Values.store.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+        {{- if .Values.store.initContainers }}
+        {{- include "graphscope-store.tplvalues.render" (dict "value" .Values.store.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      containers:
+        - name: engine
+          image: {{ include "graphscope-store.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command:
+          {{- if (eq "Vineyard" .Values.storageType) }}
+            - /bin/bash
+            - -c
+            - |
+              sleep 5s
+
+              chmod +x $GRAPHSCOPE_HOME/bin/kube_ssh
+              sudo cp $GRAPHSCOPE_HOME/bin/kube_ssh /usr/bin/kube_ssh
+
+              hosts=/tmp/hosts_of_nodes
+              rm -rf $hosts && touch $hosts
+              pod_ips=`kubectl get pod -lapp.kubernetes.io/component=store -o jsonpath='{.items[*].status.podIP}'`
+              pod_names=""
+              for ip in `echo $pod_ips`
+              do
+                pod_name=`kubectl get pods -o custom-columns=:metadata.name --no-headers=true --field-selector status.podIP=$ip`
+                echo $ip $pod_name >> $hosts
+                pod_names="${pod_names},${pod_name}"
+              done
+              pod_names=${pod_names:1}
```

Comment:
Since this is a statefulset, all pods name could be known in advance, and you doesn't need the kubectl to get it. And without using the kubectl to get the status, you could remove the serviceaccount.yaml .